nncf/common/utils/backend.py

```diff
@@ -20,8 +22,10 @@
     import openvino  # type: ignore # noqa: F401
 
     _OPENVINO_AVAILABLE = True
+    _OPENVINO_VERSION = version.parse(openvino.__version__.split("-")[0])
 except ImportError:
     _OPENVINO_AVAILABLE = False
+    _OPENVINO_VERSION = version.parse("0.0")
```

Comment:
IMHO: I don't understand why this line of code is needed...

Reply:
This is in case somebody calls `is_openvino_at_least` while openvino is not installed

Reply:
```
if not _OPENVINO_AVAILABLE:
        return False
```

Reply:
Removed